includes/contact_component.inc

```diff
@@ -238,6 +239,15 @@ function _webform_edit_civicrm_contact($component) {
       '#parents' => array('extra', 'default_relationship'),
     );
   }
+  $form['defaults']['default']['#options']['custom_ref'] = t('Custom reference field');
+  $form['defaults']['default_custom_ref'] = array(
+    '#type' => 'select',
+    '#title' => t('Specify Custom Field(s)'),
```

Comment:
I believe the label "Field(s)" is incorrect. This is a select element not a multiselect, so you cannot pick more than one.